numba/np/random/old_distributions.py

```diff
@@ -565,7 +565,7 @@ def random_binomial_btpe(bitgen, n, p):
     q = 1.0 - r
     fm = n * r + r
     m = int(np.floor(fm))
-    p1 = int(np.floor(2.195 * np.sqrt(n * r * q) - 4.6 * q) + 0.5)
+    p1 = np.floor(2.195 * np.sqrt(n * r * q) - 4.6 * q) + 0.5
```

Comment:
I confirmed this change against: https://github.com/numpy/numpy/blob/aab644b3ceb8742e2091c8cc51ff932e14b0627e/numpy/random/src/distributions/distributions.c#L635